.github/workflows/build.yml

```diff
@@ -124,6 +124,9 @@ jobs:
           - name: Windows SteamVR
             targetPlatform: StandaloneWindows64
             vrsdk: SteamVR
+          - name: Windows Pimax
```

Comment:
BTW, note that there will be a slight conflict when merging this into the xr_v2 branch, but it should be easy enough to fix

Reply:
Yup, would have preferred to target that branch but gotta be stable! Will fix in due course.